### PR TITLE
Add example remove_scalar_bar( )

### PR DESCRIPTION
**Pull request by ENIAC-6**
Add example for remove_scalar_bar( ) for plotting module to address issue #1629.

### Overview

Improve example coverage of plotting module. 

Issue #1629

### Details

Add example for remove_scalar_bar( ) to address issue #1629.

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -3809,9 +3809,19 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
 
     @wraps(ScalarBars.remove_scalar_bar)
     def remove_scalar_bar(self, *args, **kwargs):
-        """Remove the active scalar bar."""
+        """Remove the active scalar bar.
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3812]: docstring needs to be before the python code.
```suggestion
        """Remove the active scalar bar.
        
        Examples
        --------
        
        Remove scalar bar from a plotter.
        >>> p = pv.Plotter()
        >>> p.add_mesh(mesh, cmap='coolwarm')
        >>> p.remove_scalar_bar()
        >>> p.show()
        """
```

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -3809,9 +3809,19 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
...
+        Examples
+        --------
+        
+        Remove scalar bar from a plotter.
+        >>> p = pv.Plotter()
+        >>> p.add_mesh(mesh, cmap='coolwarm')
+        >>> p.remove_scalar_bar()
+        >>> p.show()
+        """
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3824]:
```suggestion

```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Nice try! I suggest the place of the docstring.

File: pyvista/plotting/plotting.py (hunk 1 above)
Comment (akaszynski) [reply pyvista/plotting/plotting.py:3812]: This also needs to be moved to `ScalarBars`. I'm moving now...

File: pyvista/plotting/scalar_bars.py [hunk 3]
```diff
@@ -81,6 +81,19 @@ def remove_scalar_bar(self, title=None, render=True):
             stop the render window from rendering when an scalar bar
             is removed.
 
+
```
Comment (akaszynski) [inline pyvista/plotting/scalar_bars.py:84]:
```suggestion

```

File: pyvista/plotting/scalar_bars.py [hunk 4]
```diff
@@ -80,6 +80,17 @@ def remove_scalar_bar(self, title=None, render=True):
             Render upon scalar bar removal.  Set this to ``False`` to
             stop the render window from rendering when an scalar bar
             is removed.
+        Examples
```
Comment (akaszynski) [inline pyvista/plotting/scalar_bars.py:83]:
```suggestion
            is removed.

        Examples
```

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks for adding this!

**Pull request merged**